code/game/gamemodes/cult/cult_items.dm

```diff
@@ -33,6 +33,11 @@
 		to_chat(user, "<span class='warning'>An overwhelming sense of nausea overpowers you!</span>")
 		user.Dizzy(120)
 
+	if(HULK in user.mutations)
+		to_chat(user, "<span class='danger'>\"you can't seem to hold the blade properly!\"</span>")

```

Comment:
with the `"` it seems as though the God is saying that, but I don't think this is a God thing?

Also the `You` should be capitalised?